test/model/create.test.ts

```diff
@@ -7,25 +7,40 @@ test('creates a new entity', () => {
   const db = factory({
     user: {
       id: primaryKey(identity(userId)),
+      name: () => 'Mark Chandler',
+      nicknames: () => ['with-heart'],
+      favoriteNumbers: () => [69, 420],
     },
   })
 
   // Without any arguments a new entity is seeded
   // using the value getters defined in the model.
   const randomUser = db.user.create()
   expect(randomUser).toHaveProperty('id', userId)
+  expect(randomUser).toHaveProperty('name', 'Mark Chandler')
+  expect(randomUser).toHaveProperty('nicknames', ['with-heart'])
+  expect(randomUser).toHaveProperty('favoriteNumbers', [69, 420])
 })
 
 test('creates a new entity with initial values', () => {
   const db = factory({
     user: {
       id: primaryKey(datatype.uuid),
+      name: String,
+      nicknames: () => [] as string[],
+      favoriteNumbers: () => [] as number[],
```

Comment:
Is this fine? I'm not sure how users could define empty primitive arrays on models without specifying the type.

Reply:
That's a tricky question. Basically, the library infers whichever value type you give it. Since an empty array is likely result in `never[]`, I think the only way to denote a specific array item type without explicitly providing initial values is to use the type assertion (`as`) here.